bindings/libdnf/rpm.i

```diff
@@ -49,8 +49,8 @@
 %include "libdnf/rpm/package_set_iterator.hpp"
 %include "libdnf/rpm/package_set.hpp"
 
-%ignore libdnf::rpm::SolvQuery::SolvQuery(SolvQuery && src);
-%include "libdnf/rpm/solv_query.hpp"
+%ignore libdnf::rpm::SolvQuery::PackageQuery(PackageQuery && src);
```

Comment:
Somehow the name space looks incorrect. Please could you verify it?

Reply:
Thanks for catching this, it's most likely incorrect. Let me check and fix it.